src/configuration/Options.cpp

```diff
@@ -58,6 +58,7 @@ void Options::initializeDefaultValues()
     _intOptions[VERBOSITY] = 2;
     _intOptions[TIMEOUT] = 0;
     _intOptions[CONSTRAINT_VIOLATION_THRESHOLD] = 20;
+    _intOptions[NUMBER_OF_SIMULATIONS] = 10;
```

Comment:
Maybe set this to 1 for now? We can submit another PR after evaluating what is the best number of simulations.

Reply:
done.